selenium-system-test/src/test/scala/hts/steps/VerifyLinksSteps.scala

```diff
@@ -0,0 +1,94 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package hts.steps
+
+import hts.browser.Browser
+import hts.pages._
+import hts.pages.registrationPages._
+import hts.utils.ScenarioContext
+
+class VerifyLinksSteps extends Steps {
+
+  Given("""^that users are authenticated$""") { () ⇒
+    AuthorityWizardPage.authenticateUser(AboutPage.expectedURL, 200, "Strong", ScenarioContext.generateEligibleNINO())
+  }
+
+  When("""^they are at the start of the hts pages""") { () ⇒
+    Browser.checkCurrentPageIs(AboutPage)
+  }
+
+  Then("""^they see all feedback, get-help and privacy links are working as they go through the journey$""") { () ⇒
+    Browser.checkCurrentPageIs(AboutPage)
+    checkForLinksThatExistOnEveryPage(AboutPage)
+    Browser.nextPage()
+
+    Browser.checkCurrentPageIs(EligibilityInfoPage)
+    checkForLinksThatExistOnEveryPage(EligibilityInfoPage)
+    Browser.nextPage()
+
+    Browser.checkCurrentPageIs(HowTheAccountWorksPage)
+    checkForLinksThatExistOnEveryPage(HowTheAccountWorksPage)
+    Browser.nextPage()
+
+    Browser.checkCurrentPageIs(HowWeCalculateBonusesPage)
+    checkForLinksThatExistOnEveryPage(HowWeCalculateBonusesPage)
+    Browser.nextPage()
+
+    Browser.checkCurrentPageIs(ApplyPage)
+    checkForLinksThatExistOnEveryPage(ApplyPage)
+
+    ApplyPage.clickStartNow()
+
+    Browser.checkCurrentPageIs(EligiblePage)
+    checkForLinksThatExistOnEveryPage(EligiblePage)
+
+    EligiblePage.clickConfirmAndContinue()
+    checkForLinksThatExistOnEveryPage(SelectEmailPage)
+
+    //try to change the email and verify links
+    SelectEmailPage.setAndVerifyNewEmail("newemail@mail.com")
+    Browser.checkCurrentPageIs(VerifyYourEmailPage)
+    checkForLinksThatExistOnEveryPage(SelectEmailPage)
+
+    //go back to original select email page and continue
+    SelectEmailPage.selectGGEmail()
+    //SelectEmailPage.clickContinue()
+    checkForLinksThatExistOnEveryPage(CreateAccountPage)
+
+    CreateAccountPage.createAccount()
+
+  }
+
+  private def checkForLinksThatExistOnEveryPage(currentPage: Page): Unit = {
+
+    //feedback link
+    Browser.isTextOnPage("BETA") shouldBe true
+    Browser.clickButtonByIdOnceClickable("feedback-link")
+    driver.navigate().back()
+
+    //Get Help with this page
+    Browser.isTextOnPage("Is there anything wrong with this page") shouldBe true
+    Browser.clickButtonByIdOnceClickable("get-help-action")
+
+    //Privacy policy
+    Browser.isTextOnPage("Privacy policy") shouldBe true
+    Browser.click on Browser.linkText("Privacy policy")
```

Comment:
again, we should check that we're actually on the privacy policy page before we redirect to `currentPage`